### ISSUE TITLE
Broken pipe errors on nodes during one node shutdown

### ISSUE THREAD

**Issue by soyacz** [scylladb/scylla#11077]
## Installation details

Kernel Version: 5.15.0-1014-azure
Scylla version (or git commit hash): `5.1.dev-20220716.1cb64de8d8ff` with build-id `4e3c06e780123c59ec3bf76b4c8710fd6ffd4b5a`
Cluster size: 6 nodes (Standard_L8s_v2)

Scylla Nodes used in this run:
- longevity-10gb-3h-master-db-node-6913aa76-eastus-6 (20.124.23.224 | 10.0.0.10) (shards: 7)
- longevity-10gb-3h-master-db-node-6913aa76-eastus-5 (20.124.23.207 | 10.0.0.9) (shards: 7)
- longevity-10gb-3h-master-db-node-6913aa76-eastus-4 (20.124.23.173 | 10.0.0.8) (shards: 7)
- longevity-10gb-3h-master-db-node-6913aa76-eastus-3 (20.124.23.96 | 10.0.0.7) (shards: 7)
- longevity-10gb-3h-master-db-node-6913aa76-eastus-2 (20.124.17.46 | 10.0.0.6) (shards: 7)
- longevity-10gb-3h-master-db-node-6913aa76-eastus-1 (20.121.12.138 | 10.0.0.5) (shards: 7)

OS / Image: `/subscriptions/6c268694-47ab-43ab-b306-3c5514bc4112/resourceGroups/scylla-images/providers/Microsoft.Compute/images/ScyllaDB-5.1.dev-0.20220716.1cb64de8d8ff-1-build-200` (azure: eastus)

Test: `longevity-10gb-3h-azure-test`
Test id: `6913aa76-920f-4686-9459-ff7bc7b96c88`
Test name: `scylla-master/longevity/longevity-10gb-3h-azure-test`
Test config file(s):

- [longevity-10gb-3h.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/cf4fd9dfa4e832d024be1c5b4ea304585d1c5db1/test-cases/longevity/longevity-10gb-3h.yaml)

## Issue description
During restart of one node (longevity-10gb-3h-master-db-node-eastus-3) other nodes got into some error state. Example from one node:
```
Jul 16 09:28:55 longevity-10gb-3h-master-db-node-eastus-5 scylla[31751]:  [shard 0] gossip - Got shutdown message from 10.0.0.7, received_generation=1657960063, local_generation=1657960063
Jul 16 09:28:55 longevity-10gb-3h-master-db-node-eastus-5 scylla[31751]:  [shard 0] gossip - InetAddress 10.0.0.7 is now DOWN, status = shutdown
Jul 16 09:28:55 longevity-10gb-3h-master-db-node-eastus-5 scylla[31751]:  [shard 5] seastar - Exceptional future ignored: std::system_error (error system:32, sendmsg: Broken pipe), backtrace: 0x4ec403e 0x4ec4530 0x4ec4838 0x4afc745 0x4ea15b6 0x4b3df94 0x4b3f377 0x4b5e3a5 0x4b1200a /opt/scylladb/libreloc/libpthread.so.0+0x92a4 /opt/scylladb/libreloc/libc.so.6+0x100322
                                                                            --------
                                                                            seastar::continuation<seastar::internal::promise_base_with_type<void>, seastar::rpc::connection::send(seastar::rpc::snd_buf, std::optional<std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > > >, seastar::rpc::cancellable*)::$_5::operator()()::{lambda(auto:1)#1}, seastar::future<void>::then_wrapped_nrvo<void, seastar::rpc::connection::send(seastar::rpc::snd_buf, std::optional<std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > > >, seastar::rpc::cancellable*)::$_5::operator()()::{lambda(auto:1)#1}>(seastar::rpc::connection::send(seastar::rpc::snd_buf, std::optional<std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > > >, seastar::rpc::cancellable*)::$_5::operator()()::{lambda(auto:1)#1}&&)::{lambda(seastar::internal::promise_base_with_type<void>&&, seastar::rpc::connection::send(seastar::rpc::snd_buf, std::optional<std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > > >, seastar::rpc::cancellable*)::$_5::operator()()::{lambda(auto:1)#1}&, seastar::future_state<seastar::internal::monostate>&&)#1}, void>
                                                                            --------
                                                                            seastar::internal::when_all_state_component<seastar::future<void> >
                                                                            --------
                                                                            seastar::continuation<seastar::internal::promise_base_with_type<void>, seastar::rpc::send_helper<netw::serializer, netw::messaging_verb, seastar::rpc::no_wait_type, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_verb, seastar::rpc::signature<seastar::rpc::no_wait_type (frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce>)>)::shelper::send(seastar::rpc::client&, std::optional<std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > > >, seastar::rpc::cancellable*, frozen_mutation const&, utils::small_vector<gms::inet_address, 3ul> const&, gms::inet_address const&, unsigned int const&, unsigned long const&, std::optional<tracing::trace_info> const&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> const&)::{lambda(auto:1)#1}, seastar::future<std::tuple<seastar::future<void>, seastar::future<void> > >::then_impl_nrvo<seastar::rpc::send_helper<netw::serializer, netw::messaging_verb, seastar::rpc::no_wait_type, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_verb, seastar::rpc::signature<seastar::rpc::no_wait_type (frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce>)>)::shelper::send(seastar::rpc::client&, std::optional<std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > > >, seastar::rpc::cancellable*, frozen_mutation const&, utils::small_vector<gms::inet_address, 3ul> const&, gms::inet_address const&, unsigned int const&, unsigned long const&, std::optional<tracing::trace_info> const&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> const&)::{lambda(auto:1)#1}, seastar::future<void> >(seastar::rpc::send_helper<netw::serializer, netw::messaging_verb, seastar::rpc::no_wait_type, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_verb, seastar::rpc::signature<seastar::rpc::no_wait_type (frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce>)>)::shelper::send(seastar::rpc::client&, std::optional<std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > > >, seastar::rpc::cancellable*, frozen_mutation const&, utils::small_vector<gms::inet_address, 3ul> const&, gms::inet_address const&, unsigned int const&, unsigned long const&, std::optional<tracing::trace_info> const&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> const&)::{lambda(auto:1)#1}&&)::{lambda(seastar::internal::promise_base_with_type<void>&&, seastar::rpc::send_helper<netw::serializer, netw::messaging_verb, seastar::rpc::no_wait_type, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(auto:2, seastar::rpc::signature<auto:3 (auto:4, auto:4, auto:4, auto:4, auto:4, auto:4, auto:4)>)::shelper::send(seastar::rpc::client&, std::optional<std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > > >, seastar::rpc::cancellable*, frozen_mutation const&, utils::small_vector<gms::inet_address, 3ul> const&, gms::inet_address const&, unsigned int const&, unsigned long const&, std::optional<tracing::trace_info> const&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> const&)::{lambda(auto:1)#1}&, seastar::future_state<std::tuple<seastar::future<void>, seastar::future<void> > >&&)#1}, std::tuple<seastar::future<void>, seastar::future<void> > >
                                                                            --------
                                                                            seastar::continuation<seastar::internal::promise_base_with_type<void>, seastar::future<void>::handle_exception<netw::send_message_timeout<seastar::rpc::no_wait_type, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_service*, netw::messaging_verb, netw::msg_addr, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation&&, utils::small_vector<gms::inet_address, 3ul>&&, gms::inet_address&&, unsigned int&&, unsigned long&&, std::optional<tracing::trace_info>&&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce>&&)::{lambda(std::__exception_ptr::exception_ptr&&)#1}>(netw::send_message_timeout<seastar::rpc::no_wait_type, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_service*, netw::messaging_verb, netw::msg_addr, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation&&, utils::small_vector<gms::inet_address, 3ul>&&, gms::inet_address&&, unsigned int&&, unsigned long&&, std::optional<tracing::trace_info>&&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce>&&)::{lambda(std::__exception_ptr::exception_ptr&&)#1}&&)::{lambda(auto:1&&)#1}, seastar::future<void>::then_wrapped_nrvo<seastar::future<void>, seastar::future<void>::handle_exception<netw::send_message_timeout<seastar::rpc::no_wait_type, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_service*, netw::messaging_verb, netw::msg_addr, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation&&, utils::small_vector<gms::inet_address, 3ul>&&, gms::inet_address&&, unsigned int&&, unsigned long&&, std::optional<tracing::trace_info>&&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce>&&)::{lambda(std::__exception_ptr::exception_ptr&&)#1}>(netw::send_message_timeout<seastar::rpc::no_wait_type, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_service*, netw::messaging_verb, netw::msg_addr, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation&&, utils::small_vector<gms::inet_address, 3ul>&&, gms::inet_address&&, unsigned int&&, unsigned long&&, std::optional<tracing::trace_info>&&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce>&&)::{lambda(std::__exception_ptr::exception_ptr&&)#1}&&)::{lambda(auto:1&&)#1}>(seastar::future<void>::handle_exception<netw::send_message_timeout<seastar::rpc::no_wait_type, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_service*, netw::messaging_verb, netw::msg_addr, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation&&, utils::small_vector<gms::inet_address, 3ul>&&, gms::inet_address&&, unsigned int&&, unsigned long&&, std::optional<tracing::trace_info>&&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce>&&)::{lambda(std::__exception_ptr::exception_ptr&&)#1}>(netw::send_message_timeout<seastar::rpc::no_wait_type, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_service*, netw::messaging_verb, netw::msg_addr, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation&&, utils::small_vector<gms::inet_address, 3ul>&&, gms::inet_address&&, unsigned int&&, unsigned long&&, std::optional<tracing::trace_info>&&, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce>&&)::{lambda(std::__exception_ptr::exception_ptr&&)#1}&&)::{lambda(auto:1&&)#1}&&)::{lambda(seastar::internal::promise_base_with_type<void>&&, seastar::future<void>::handle_exception<netw::send_message_timeout<seastar::rpc::no_wait_type, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >, frozen_mutation, utils::small_vector<gms::inet_address, 3ul>, gms::inet_address, unsigned int, unsigned long, std::optional<tracing::trace_info>, std::variant<std::monostate, db::per_partition_rate_limit::account_only, db::per_partition_rate_limit::account_and_enforce> >(netw::messaging_service*, netw::messaging_verb, netw::msg_addr, auto:2, auto:3&&, auto:3&&, auto:3&&, auto:3&&, auto:3&&, auto:3&&, auto:3&&)::{lambda(std::__exception_ptr::exception_ptr&&)#1}>(auto:1&&)::{lambda(auto:1&&)#1}&, seastar::future_state<seastar::internal::monostate>&&)#1}, void>
                                                                            --------
                                                                            seastar::continuation<seastar::internal::promise_base_with_type<void>, seastar::future<void>::finally_body<service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_30::operator()(gms::inet_address, utils::small_vector<gms::inet_address, 3ul>&&) const::{lambda()#1}, false>, seastar::future<void>::then_wrapped_nrvo<seastar::future<void>, seastar::future<void>::finally_body<service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_30::operator()(gms::inet_address, utils::small_vector<gms::inet_address, 3ul>&&) const::{lambda()#1}, false> >(seastar::future<void>::finally_body<service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_30::operator()(gms::inet_address, utils::small_vector<gms::inet_address, 3ul>&&) const::{lambda()#1}, false>&&)::{lambda(seastar::internal::promise_base_with_type<void>&&, seastar::future<void>::finally_body<service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_30::operator()(gms::inet_address, utils::small_vector<gms::inet_address, 3ul>&&) const::{lambda()#1}, false>&, seastar::future_state<seastar::internal::monostate>&&)#1}, void>
                                                                            --------
                                                                            seastar::continuation<seastar::internal::promise_base_with_type<void>, seastar::future<void>::handle_exception<service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_98>(service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_98&&)::{lambda(auto:1&&)#1}, seastar::future<void>::then_wrapped_nrvo<seastar::future<void>, seastar::future<void>::handle_exception<service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_98>(service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_98&&)::{lambda(auto:1&&)#1}>(seastar::future<void>::handle_exception<service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_98>(service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_98&&)::{lambda(auto:1&&)#1}&&)::{lambda(seastar::internal::promise_base_with_type<void>&&, seastar::future<void>::handle_exception<service::storage_proxy::send_to_live_endpoints(unsigned long, std::chrono::time_point<seastar::lowres_clock, std::chrono::duration<long, std::ratio<1l, 1000000000l> > >)::$_98>(auto:1&&)::{lambda(auto:1&&)#1}&, seastar::future_state<seastar::internal::monostate>&&)#1}, void>
```
I'm not sure if it is related - backend is Azure.

- Restore Monitor Stack command: `$ hydra investigate show-monitor 6913aa76-920f-4686-9459-ff7bc7b96c88`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=6913aa76-920f-4686-9459-ff7bc7b96c88)
- Show all stored logs command: `$ hydra investigate show-logs 6913aa76-920f-4686-9459-ff7bc7b96c88`

## Logs:
- **db-cluster-6913aa76.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/db-cluster-6913aa76.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/db-cluster-6913aa76.tar.gz)
- **monitor-set-6913aa76.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/monitor-set-6913aa76.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/monitor-set-6913aa76.tar.gz)
- **loader-set-6913aa76.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/loader-set-6913aa76.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/loader-set-6913aa76.tar.gz)
- **sct-runner-6913aa76.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/sct-runner-6913aa76.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/sct-runner-6913aa76.tar.gz)
- **parallel-timelines-report-6913aa76.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/parallel-timelines-report-6913aa76.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/6913aa76-920f-4686-9459-ff7bc7b96c88/20220716_112344/parallel-timelines-report-6913aa76.tar.gz)

[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/longevity-10gb-3h-azure-test/53/)

**Comment by bhalevy** [comment]
Saw this also in https://github.com/scylladb/scylla/issues/11063,
though there, the issue is about the storage_proxy spamming the log.

```
Jul 16 09:28:55 longevity-10gb-3h-master-db-node-eastus-5 scylla[31751]:  [shard 5] seastar - Exceptional future ignored: std::system_error (error system:32, sendmsg: Broken pipe), backtrace: 0x4ec403e 0x4ec4530 0x4ec4838 0x4afc745 0x4ea15b6 0x4b3df94 0x4b3f377 0x4b5e3a5 0x4b1200a /opt/scylladb/libreloc/libpthread.so.0+0x92a4 /opt/scylladb/libreloc/libc.so.6+0x100322
```

This is https://github.com/scylladb/seastar/issues/1145
Regression was caused by https://github.com/scylladb/seastar/commit/c437920c23aac97d30fdba1e1c6554f15d98eb5a
and fixed by https://github.com/scylladb/seastar/commit/1d4432ed281ef3c610c4ef17968f15358a1d6755

https://github.com/scylladb/scylla/pull/11082 pulls this fix to scylla.

In any case `seastar - Exceptional future ignored` is a warning and in most cases it is benign, so there's no justification for `master/high`

**Comment by bhalevy** [comment]
Fixed by b6273fee39dda92b5715d1a4ba533c93ea5328c7 /
scylladb/seastar@1d4432ed281ef3c610c4ef17968f15358a1d6755

**Comment by bhalevy** [comment]
no backport needed as this was caused by https://github.com/scylladb/seastar/commit/c437920c23aac97d30fdba1e1c6554f15d98eb5a that is confined to master.
